.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,96 @@
-blank_issues_enabled: true
-contact_links:
-  - name: I have some questions about Flame.
-    url: https://stackoverflow.com/tags/flame
-    about: Ask your questions on StackOverflow! The community is always willing to help out.
-  - name: Join us on Discord!
-    url: https://discord.gg/pxrBmy4
-    about: Ask your questions in our community!
+name: Bug Report
```

Comment:
NIT: I'm not sure if this is the changeset we want; the change of the markdown looks solid 🙇 But I don't believ we should be modifying the `config.yml` but instead be creating a new file, `bug_report.yml` for example, in this same folder with the contents you've added (each new template/form should be it's own unique file).

More info here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

Reply:
Hi @anleac does it look good now?